cookbooks/env.rst

```diff
@@ -203,6 +203,17 @@ will evaluate to an empty string like any other unset variable:
     When several routes match a rule, the first one wins, the user order is
     kept. There's no preference regarding protocols.
 
+  .. tip::
+
+    For complex multi-applications projects where several applications are
+    publicly reachable, but one needs to determine the current application
+    endpoint (for webhooks for examples) and the project endpoint (to send
+    emails for instance), an additional
+    ``SYMFONY_APPLICATION_DEFAULT_ROUTE_`` environment variables set is
```

Comment:
I would add * at the end `SYMFONY_APPLICATION_DEFAULT_ROUTE_*`

Reply:
👍